packages/react-components/src/components/DomainPermissions.tsx

```diff
@@ -58,7 +59,9 @@ const _DomainPermissionsContainer = (props: DomainPermissionsProps): JSX.Element
           <Icon styles={iconPrimaryStyles} iconName={'ControlButtonCameraOn'}></Icon>
           <Icon styles={iconBackDropStyles} iconName={'iconBackdrop'}></Icon>
         </Stack>
-        <Icon styles={iconPrimaryStyles} iconName={'Sparkle'}></Icon>
```

Comment:
This should be its own PR - "updating to use sparkle icon" - this PR should _just_ be the new icon package

Reply:
this PR: https://github.com/Azure/communication-ui-library/pull/2307 😊